app/routes/org-group-profile.js

```diff
@@ -0,0 +1,11 @@
+import Route from '@ember/routing/route';
+
+
+
```

Comment:
Codacy found an issue: [More than 2 blank lines not allowed.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6000753)